lvmsnapshotter.go

```diff
@@ -350,10 +360,12 @@ func (o *snapshotter) mounts(s storage.Snapshot) []mount.Mount {
 // Close closes the snapshotter
 func (o *snapshotter) Close() error {
 	llog.Printf("Close function called")
-	cmd := "umount"
-	args := []string{metaVolumeMountPath}
-	_, _ = runCommand(cmd, args)
-	return o.ms.Close()
+	var err error
```

Comment:
@ganeshmaharaj I assume that any running containers will continue to work. 

So in  can stop the snapshotter. Update it. Start it back up and things continue on?

Reply:
Who will call close? Should you call an explicit close on your own on graceful stop?

Reply:
This gets called when rpc.GracefulStop() is called. I believe that the running containers will still be usable as we are just killing the rpc servicer and not the lvms themselves.